src/cc65/main.c

```diff
@@ -951,6 +962,9 @@ int main (int argc, char* argv[])
 
                 case 'O':
                     IS_Set (&Optimize, 1);
+                    IS_Set (&CodeSizeFactor, 100);
+                    IS_Set (&EnableRegVars, 0);
+                    IS_Set (&InlineStdFuncs, 0);
```

Comment:
but this is different... -O2 and -O0 are optimization levels, which enable/disable a whole lot of seperate options. -Os and -Oi are not like that, those enable detailed options, and should add up on the commandline, IMHO